cypress/integration/tests/controls/stakeholdergroups/create.test.ts

```diff
@@ -1,35 +1,68 @@
 /// <reference types="cypress" />
 
-import { login } from "../../../../utils/utils";
-import { Stakeholdergroups } from "../../../models/stakeholdergroups";
-import { tdTag } from "../../../types/constants";
+import { login, clickByText, inputText } from "../../../../utils/utils";
+import { navMenu, navTab } from "../../../views/menu.view";
+import { controls, stakeholdergroups } from "../../../types/constants";
+import {
+    stakeholdergroupNameInput,
+    stakeholdergroupNameHelper,
+    stakeholdergroupDescriptionInput,
+    stakeholdergroupDescriptionHelper,
+} from "../../../views/stakeholdergroups.view";
 
-describe("A single Stakeholder group", () => {
-    const stakeholdergroup = new Stakeholdergroups();
+import * as commonView from "../../../../integration/views/commoncontrols.view";
+import * as faker from "faker";
 
-    before("Login", function () {
+describe("Basic checks while creating stakeholder groups", () => {
+    beforeEach("Login", function () {
         // Perform login
         login();
 
-        // Interceptors
-        cy.intercept("POST", "/api/controls/stakeholder-group*").as("postStakeholdergroups");
-        cy.intercept("GET", "/api/controls/stakeholder-group*").as("getStakeholdergroups");
+        // Navigate to "New stakeholder group" page
+        clickByText(navMenu, controls);
+        clickByText(navTab, stakeholdergroups);
+        clickByText("button", "Create new");
     });
 
-    it("Stakeholder group crud operations", function () {
-        // Create new stakeholder group
-        stakeholdergroup.create();
-        cy.wait("@postStakeholdergroups");
+    it("Stakeholder name and description contraints check", function () {
```

Comment:
These tests are good, but not sure the file name create is suitable. As per earlier discussion, crud tests are to be put in create.test.ts

spell check - constraints